lib/ex_unit/lib/ex_unit/formatter.ex

```diff
@@ -174,14 +178,22 @@ defmodule ExUnit.Formatter do
   """
   def format_test_all_failure(test_module, failures, counter, width, formatter) do
     name = test_module.name
-    test_case_info(with_counter(counter, "#{inspect name}: "), formatter) <>
+
+    test_case_info(with_counter(counter, "#{inspect(name)}: "), formatter) <>
       Enum.map_join(Enum.with_index(failures), "", fn {{kind, reason, stack}, index} ->
         {text, stack} = format_kind_reason(test_module, kind, reason, stack, width, formatter)
         failure_header(failures, index) <> text <> format_stacktrace(stack, name, nil, formatter)
       end)
   end
 
-  defp format_kind_reason(test, :error, %ExUnit.AssertionError{} = struct, stack, width, formatter) do
+  defp format_kind_reason(
```

Comment:
This looks out of place to have a multi-line function head, with a single line function.  Suggestions?

Reply:
In this case it's fine since there's no straightforward way to avoid it (we'd have to maybe make the arguments into a map or something like this but let's keep that for after the formatting craze is over).